dotCMS/src/integration-test/java/com/dotcms/integritycheckers/IntegrityUtilTest.java

```diff
@@ -0,0 +1,463 @@
+import java.util.List;
+import java.util.Map;
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.lang.StringUtils;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class IntegrityUtilTest  {
+
+    @Before
+    public void setup() throws Exception {
+        IntegrationTestInitService.getInstance().init();
+    }
+
+    /**
+     * Method to test: {@link IntegrityUtil#completeCheckIntegrity(String)}
+     * When: There is a folder with the same path but different inode ad identifier
+     * Should: Insert a register in folders_ir
+     * @throws Exception
+     */
+    @Test
+    public void folderWithConflicts() throws Exception {
+        final Host host = new SiteDataGen().nextPersisted();
+        final Folder folderParent = new FolderDataGen()
+                .site(host)
+                .name("parent_" + System.currentTimeMillis())
+                .nextPersisted();
+
+        final Folder folder = new FolderDataGen()
+                .site(host)
+                .parent(folderParent)
+                .nextPersisted();
+
+        final Identifier identifier = APILocator.getIdentifierAPI().find(folder.getIdentifier());
+        final String endPointID = "endpointID_" + System.currentTimeMillis();
+
+        final String path = createCSVFile(endPointID).get(IntegrityType.FOLDERS);
+
+        final CsvWriter foldersCsvWriter = new CsvWriter(path, '|', StandardCharsets.UTF_8);
+        final long currentTimeMillis = System.currentTimeMillis();
+        final String newInode_1 = String.valueOf(currentTimeMillis);
+        final String newIdentifier_1 = String.valueOf(currentTimeMillis + 1);
+
+        foldersCsvWriter.write(newInode_1);
+        foldersCsvWriter.write(newIdentifier_1);
+        foldersCsvWriter.write(identifier.getParentPath());
+        foldersCsvWriter.write(folder.getName());
+        foldersCsvWriter.write(folder.getHostId());
+        foldersCsvWriter.endRecord();
+
+        final String newInode_2 = String.valueOf(currentTimeMillis + 2);
+        final String newIdentifier_2 = String.valueOf(currentTimeMillis + 3);
+
+        foldersCsvWriter.write(newInode_2);
+        foldersCsvWriter.write(newIdentifier_2);
+        foldersCsvWriter.write("/any-path");
+        foldersCsvWriter.write("folder-name");
+        foldersCsvWriter.write(folder.getHostId());
+        foldersCsvWriter.endRecord();
+
+        foldersCsvWriter.flush();
+        foldersCsvWriter.close();
+
+        IntegrityUtil.completeCheckIntegrity(endPointID);
+
+        DotConnect dc = new DotConnect();
+        dc.setSQL("select * from folders_ir where endpoint_id = ?");
+        dc.addObject(endPointID);
+        final List<Map<String, Object>> results = dc.loadObjectResults();
+
+        assertEquals(1, results.size());
+
+        assertEquals(folder.getIdentifier(), results.get(0).get("local_inode"));
+        assertEquals(folder.getInode(), results.get(0).get("local_identifier"));
```

Comment:
Same here. Looks like it should be 'folder.getIdentifier()'

Reply:
done https://github.com/dotCMS/core/pull/21743/commits/06e0b7fd1a20f8d94d51bbb98ee2f99262306ff1#diff-955f4199431261a5a18c844f03e72d5d53a98da73e62f16374c76aaa41a611fdL109